frameworks/Java/blade/benchmark_config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "framework": "blade",
+  "tests": [{
+    "default": {
+      "setup_file": "setup",
+      "json_url": "/json",
+      "db_url": "/db",
+      "query_url": "/db?queries=",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "MySQL",
+      "framework": "blade",
+      "language": "Java",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "blade",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "blade-mvc",
```

Comment:
Are there other variants to `blade` other than `mvc`? If not, I think this should be change to `blade`.

Reply:
There are some other components, such as the expansion of the session, extending the template engine, but I'm not sure whether to modify it.